docs/en/enterprise-edition/policy-reference/azure-policies/azure-networking-policies/bc-azure-2-49.adoc

```diff
@@ -0,0 +1,53 @@
+== Azure Machine learning workspace configured with overly permissive network access
+
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+| 76b345ae-d738-4485-8cfc-ed0c265ea016
+
+|Checkov ID
+| https://github.com/bridgecrewio/checkov/blob/main/checkov/terraform/checks/graph_checks/azure/AzureMLWorkspacePublicNetwork.yaml[CKV2_AZURE_49]
+
+|Severity
+|HIGH
+
+|Subtype
+|Run,Build
+
+|Frameworks
+|Terraform,TerraformPlan,ARM,Bicep
+
+|===
+
+=== Description
+
+This policy is checking to ensure that the Azure Machine Learning workspace is not configured with overly permissive network access. Overly permissive network access could allow unauthorized individuals or systems to access and potentially manipulate the workspace, leading to potential data breaches or compromises in the machine learning process. This could further lead to flawed model results, data manipulation, or unauthorized data access. Thus, it's important to verify that the network access to the Azure Machine Learning workspace is properly restricted to prevent such risks.
```

Comment:
This policy detects whether the Azure Machine Learning workspace is configured with overly permissive network access. Such unrestricted access creates security vulnerabilities. Excessive network access permissions allow individuals or systems to access and potentially manipulate the workspace,  allowing malicious actors to exploit these weaknesses to gain unauthorized access potentially leading to data breaches or manipulation of the machine learning process itself. This unauthorized access could compromise models, leading to flawed outputs or biased results. Additionally, sensitive data stored or processed within the workspace might be exfiltrated if network access isn't properly restricted.